3520/1/lib/repo_scrapper.rb

```diff
@@ -0,0 +1,67 @@
+require 'webdrivers/geckodriver'
+require 'watir'
+class RepoScrapper
+  attr_reader :repo_info
+
+  def initialize
+    @browser = Watir::Browser.new :firefox, headless: true
+    @repo_info = {}
+  end
+
+  def repo_info_parse
+    repo_gem_name
+    repo_used_by
+    repo_watch_star_forks
+    repo_contributors
+    repo_issues
+  end
+
+  def github_link
+    if @browser.link(text: 'Homepage').href.include?('github.com')
+      @browser.link(text: 'Homepage').href
+    elsif @browser.link(text: 'Source Code').href.include?('github.com')
+      @browser.link(text: 'Source Code').href
+    else
+      raise 'There is no link for github from rubygems.org'
+    end
+  end
+
+  def get_page(link)
+    @browser.goto(link)
+    sleep 0.5
+  rescue Selenium::WebDriver::Error::UnknownError
+    puts 'Check internet connection.'
+    exit
```

Comment:
why you exit from the program because of one line???

Reply:
> why you exit from the program because of one line???
Can I use abort 'message' or it's not a good choice? Also make message as a constant?

Reply:
the idea is to process failure without exiting because of one gem. so print message that gem is not processable

Reply:
and continue with other gems.

Reply:
Done.